datagenerator/src/main/resources/rythm/common_header.rythm

```diff
@@ -1 +1,7 @@
+@args String type
+
 // DO NOT EDIT - THIS FILE IS GENERATED BY restli-csharp
+
+public class @type {
```

Comment:
This may not be included in common_header.rythm, since enum and record are different here.